package.json

```diff
@@ -75,6 +75,9 @@
     "ua-parser-js": "^1.0.35",
     "zod": "^3.22.3"
   },
+  "overrides": {
+    "ws": "8.17.1"
```

Comment:
This was necessary in the end?

Which are the packages we need to watch to upgrade to be able to remove this?

Could you add it in the PR description so that we can come back to it to better understand it?

Reply:
Yes, it was. `puppeteer-core` seems to be the culprit:

```
+-- vitest@1.6.0
| `-- jsdom@20.0.3
|   `-- ws@8.17.1 overridden
`-- webdriverio@8.39.0
  +-- devtools@8.39.0
  | `-- puppeteer-core@20.3.0
  |   `-- ws@8.13.0 invalid: "8.17.1" from node_modules/devtools/node_modules/puppeteer-core overridden
  +-- puppeteer-core@20.9.0
  | `-- ws@8.13.0 invalid: "8.17.1" from node_modules/puppeteer-core overridden
  `-- webdriver@8.39.0
    `-- ws@8.17.1 deduped
```